tests/fbchat_test.py

```diff
@@ -15,9 +15,6 @@ class Object():
     text: str = ''
 
 
-# password: str = str(config.get_api_key('TESTFB', 'password'))
-# email: str = str(config.get_api_key('TESTFB', 'email'))
-
 client = fbchat_api.FacebookChatApi('ddo3@case.edu', 'seniorproject')
```

Comment:
Still here?

Reply:
No its not. Look under the files changed tab

Reply:
still there for me: https://imgur.com/a/KNxuH00

Reply:
I'm referencing the 
`client = fbchat_api.FacebookChatApi('ddo3@case.edu', 'seniorproject')`
line, because we don't want to commit private account names and passwords.

Reply:
If I don't have that line, then I can't do any of my tests. I need to have someone logged in or the client does not work. It is almost impossible to mock the initialization of the client object I need to create.

Reply:
Otherwise, I would have to use that skip thing for all of my tests

Reply:
That's probably what it has to be, because we can't / shouldn't commit our account secrets to github

If you take a look at how I mocked mine in my pull request in discord_api_test.py, you can see 1 example of how I mock all the functions that actually log into the app, and assert that they were called exactly once correctly and that the return value was correct. 

If you're in a time bind, you could also just use that skip thing, although it's risky since tests can pass/fail locally, if you do, upload a correct config file to slack